go/vt/sqlparser/ast_funcs.go

```diff
@@ -2981,3 +2981,67 @@ func ExtractAllTables(stmt Statement) []string {
 	}, stmt)
 	return tables
 }
+
+var _ SelectStatement = (*ValuesStatement)(nil)
+
+func (node *ValuesStatement) GetLock() Lock {
+	return NoLock
+}
+
+func (node *ValuesStatement) SetLock(lock Lock) {
+	if lock != NoLock {
+		panic("cannot set lock on Values statement")
```

Comment:
Hmm... This doesn't look right.

Reply:
This is used in grammar, as part of the interface. 
We can choose to ignore it if not panic